geomstats/_backend/pytorch/__init__.py

```diff
@@ -573,6 +471,6 @@ def array_from_sparse(indices, data, target_shape):
 
 def from_vector_to_diagonal_matrix(x):
```

Comment:
The function is used in the `SymmetricMatrices` class so it needs to be exposed. The question is whether it's actually needed somewhere else or if it could be turned into a method of the class instead.

Reply:
I think this function was implemented because the np.diag function was not vectorized as we needed. Let's keep it here then. Thanks!